MidwestAccessCoalition/lockbox_rails
# MFA - backend only

## Changelog
- adds dependency of devise-authy gem
- adds authy columns to users table
- adds boilerplate devise-authy code
- adds `config/local_env.yml` to the gitignore and adds a sample local_env.yml


## Link to issue:  
Part of #429 


## Steps for QA/Special Notes:
- can QA functionality manually by setting the AUTHY_MFA_ENABLED and the AUTHY_API_KEY env vars locally
- from a logged in account, can enable MFA by navigating to /users/enable_mfa and going through the set up flow
- to verify that it's working, log out and then log back in and notice that you have to go through MFA steps
- added a very lightweight system test to verify that MFA is only occuring when AUTHY_MFA_ENABLED is set to something truthy

## Relevant Screenshots: 


## Are you ready for review?:

- [ ] Added relevant screenshots
